### ISSUE TITLE
The console appears then disappears but doesn't run valorant

### ISSUE THREAD

**Issue by Fanaticgiant547** [colinhartigan/valorant-rpc#4]

**Comment by colinhartigan** [comment]
most likely installed wrong, check that you did everything correctly

**Comment by enriqueornelasjr** [comment]
I installed and this occurred to me as well. Upon running the executable from the command prompt, I got this error below. Hope this helps! (I'm running v1.2, Win 10)

![image](https://user-images.githubusercontent.com/19866752/111673086-9b252f80-87e8-11eb-935f-f29bb509ef95.png)

**Comment by colinhartigan** [comment]
try [v1.3](https://github.com/colinhartigan/valorant-rpc/releases/tag/v1.3), it removes the need to add a system environment variable and should fix your problem

**Comment by enriqueornelasjr** [comment]
Sweet! On the first time, it did not run correctly and spat an error (which I didn't catch because I ran it from my shortcut). Upon closing and re-opening, however, I worked like a charm! Upon being able to restart my computer, I will do so and output the error that I got. For now I would update the Reddit post to link v1.3, since it is what confused me and maybe other people as well. Apart from that, I'm excited to start using it! Thanks!
